opencensus/stats/internal/view_data_impl.cc

```diff
@@ -302,5 +305,60 @@ ViewDataImpl::ViewDataImpl(ViewDataImpl* source, absl::Time now)
   }
 }
 
+void ViewDataImpl::SetUpdateTime(const std::vector<std::string>& tag_values,
+                                 absl::Time now) {
+  if (expiry_duration_ == absl::ZeroDuration()) {
+    // No need to track last update time if expiry duration is not set.
+    return;
+  }
+
+  auto update_time_map_iter = update_time_entries_.find(tag_values);
```

Comment:
I'm worried about the performance of list::find. Why is this a list?

Reply:
`update_time_entries_` is a map from tags to list entry iterator. We need the list to keep track of update time of each view data. This map is used to find the list entry to update time for in constant time.

Reply:
Oops, I misread it. We keep both a map and a list, so we can early exit after processing just enough of the list (L337)?

Reply:
Yeah, the list is sorted based on last updated time, so that we don't need to scan all time series on every update.